qiskit_ibm_runtime/options/dynamical_decoupling_options.py

```diff
@@ -19,32 +19,30 @@
 
 @primitive_dataclass
 class DynamicalDecouplingOptions:
-    """Options for dynamical decoupling (DD).
-
-    Args:
-        enable: Whether to enable DD as specified by the other options in this class.
-            Default: False.
-
-        sequence_type: Which dynamical decoupling sequence to use. Default: "XX".
-
-            * ``"XX"``: use the sequence ``tau/2 - (+X) - tau - (+X) - tau/2``
-            * ``"XpXm"``: use the sequence ``tau/2 - (+X) - tau - (-X) - tau/2``
-            * ``"XY4"``: : use the sequence
-              ``tau/2 - (+X) - tau - (+Y) - tau (-X) - tau - (-Y) - tau/2``
-
-        extra_slack_distribution: Where to put extra timing delays due to rounding issues.
-            Rounding issues arise because the discrete time step ``dt`` of the system cannot
-            be divided. This option takes following values. Default: "middle".
-
-            * ``"middle"``: Put the extra slack to the interval at the middle of the sequence.
-            * ``"edges"``: Divide the extra slack as evenly as possible into intervals at
-              beginning and end of the sequence.
-
-        scheduling_method: Whether to schedule gates as soon as ("asap") or
-            as late as ("alap") possible. Default: "alap".
-    """
+    """Options for dynamical decoupling (DD)."""
 
     enable: Union[UnsetType, bool] = Unset
+    r"""Whether to enable DD as specified by the other options in this class.
+        Default: False.
+    """
     sequence_type: Union[UnsetType, Literal["XX", "XpXm", "XY4"]] = Unset
+    r"""Which dynamical decoupling sequence to use. Default: "XX".
```

Comment:
default?

I think a bunch of them are missing, so check the other docstrings

Reply:
I need to double check these all render correctly too